### ISSUE TITLE
Prefetch slides

### ISSUE THREAD

**Issue by yschroeder** [bigbluebutton/greenlight#1244]
When changing slides, especially slides with large images load slowly. The transistion between slides is via a white image. This makes it hard to follow bullet points that are showing up one after the other.

Also, with many viewers, this generates large bandwidth peaks at the server.

Greenlight should prefetch the next few slides, so the transition between them is instantly (as long as the presenter does not skip through slides too fast).

**Comment by farhatahmad** [comment]
Hi @yschroeder 

Greenlight is simply the front-end that uses BigBlueButton for conferencing. Slides are related to BigBlueButton so you'd be better off posting in that repo or in our forums.
